Jenkinsfile

```diff
@@ -8,8 +8,12 @@
 def images = [
     [image: "docker.io/library/amazonlinux:2",          arches: ["aarch64"]],
     [image: "docker.io/library/centos:7",               arches: ["amd64", "aarch64"]],
-    [image: "quay.io/centos/centos:stream8",            arches: ["amd64", "aarch64"]],
-    [image: "quay.io/centos/centos:stream9",            arches: ["amd64", "aarch64"]],
+    [image: "quay.io/centos/centos:stream8",            arches: ["amd64", "aarch64"]], // EOL: 2024-05-31
+    [image: "quay.io/centos/centos:stream9",            arches: ["amd64", "aarch64"]], // EOL: 2027
+    [image: "docker.io/library/rockylinux:8.8",         arches: ["amd64", "aarch64"]], // Rocky Linux 8 (EOL: 2029-05-31)
+    [image: "docker.io/library/rockylinux:9.2",         arches: ["amd64", "aarch64"]], // Rocky Linux 9 (EOL: 2032-05-31)
+    [image: "docker.io/library/almalinux:8.8",          arches: ["amd64", "aarch64"]], // AlmaLinux 8 (EOL: 2029)
+    [image: "docker.io/library/almalinux:9.2",          arches: ["amd64", "aarch64"]], // AlmaLinux 9 (EOL: 2032)
```

Comment:
Some questions; perhaps you're able to answer (I didn't investigate yet);

- Do rocky and alma support minor versions, or do they use a rolling release model as CentOS did (i.e. CentOS 7.2 becomes EOL once 7.3 is released).
- Do rocky and/or alma also have `$releasever` (which can vary, and may have minor version, or may have (e.g.) `8Server` etc)? Asking, because we've had a lot of issues with those (and we still haven't fully fixed that)

Reply:
> Some questions; perhaps you're able to answer (I didn't investigate yet);
> 
> * Do rocky and alma support minor versions, or do they use a rolling release model as CentOS did (i.e. CentOS 7.2 becomes EOL once 7.3 is released).

The latter one.

> * Do rocky and/or alma also have `$releasever` (which can vary, and may have minor version, or may have (e.g.) `8Server` etc)? Asking, because we've had a lot of issues with those (and we still haven't fully fixed that)

- https://dl.rockylinux.org/pub/rocky/9/: ✅  seems symlinked to `9.2`
- https://dl.rockylinux.org/pub/rocky/9.0/:  ❌ Already empty except [`README.txt`](https://dl.rockylinux.org/pub/rocky/9.0/README.txt). The contents were moved to https://dl.rockylinux.org/vault/rocky/9.0/
- https://dl.rockylinux.org/pub/rocky/9.1/:  ❌ Already empty except [`README.txt`](https://dl.rockylinux.org/pub/rocky/9.1/README.txt). The contents were moved to https://dl.rockylinux.org/vault/rocky/9.1/
- https://dl.rockylinux.org/pub/rocky/9.2/ : ✅  the latest release, as of time of writing this

The situation is same for Alma too https://repo.almalinux.org/almalinux/

Reply:
`$releasever` seems extracted to the latest revision (9.2), not to the revision of the image (9.1)

```console
$ docker run -it --rm rockylinux:9.1
[root@c19222d20aa4 /]# cat /etc/os-release 
NAME="Rocky Linux"
VERSION="9.1 (Blue Onyx)"
ID="rocky"
ID_LIKE="rhel centos fedora"
VERSION_ID="9.1"
PLATFORM_ID="platform:el9"
PRETTY_NAME="Rocky Linux 9.1 (Blue Onyx)"
ANSI_COLOR="0;32"
LOGO="fedora-logo-icon"
CPE_NAME="cpe:/o:rocky:rocky:9::baseos"
HOME_URL="https://rockylinux.org/"
BUG_REPORT_URL="https://bugs.rockylinux.org/"
ROCKY_SUPPORT_PRODUCT="Rocky-Linux-9"
ROCKY_SUPPORT_PRODUCT_VERSION="9.1"
REDHAT_SUPPORT_PRODUCT="Rocky Linux"
REDHAT_SUPPORT_PRODUCT_VERSION="9.1"

[root@c19222d20aa4 /]# cat /etc/yum.repos.d/rocky.repo 
...
[baseos]
name=Rocky Linux $releasever - BaseOS
mirrorlist=https://mirrors.rockylinux.org/mirrorlist?arch=$basearch&repo=BaseOS-$release
ver$rltype
#baseurl=http://dl.rockylinux.org/$contentdir/$releasever/BaseOS/$basearch/os/
gpgcheck=1
enabled=1
countme=1
metadata_expire=6h
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-9
...

[root@c19222d20aa4 /]# dnf -v repolist --enabled
...
Repo-id            : baseos
Repo-name          : Rocky Linux 9 - BaseOS
Repo-revision      : 1694020832
Repo-updated       : Wed Sep  6 17:20:32 2023
Repo-pkgs          : 1156
Repo-available-pkgs: 1156
Repo-size          : 1.2 G
Repo-mirrors       : https://mirrors.rockylinux.org/mirrorlist?arch=x86_64&repo=BaseOS-9
Repo-baseurl       : http://ftp.riken.jp/Linux/rocky/9.2/BaseOS/x86_64/os/ (28 more)
Repo-expire        : 21600 second(s) (last: Fri Sep 15 13:57:49 2023)
Repo-filename      : /etc/yum.repos.d/rocky.repo
...
```

Reply:
`$releasever` is expanded to `9`:
```console
$ docker run -it --rm rockylinux:9.1
[root@117abf8b612e /]# dnf install -q -y dnf-plugins-core
Importing GPG key 0x350D275D:
 Userid     : "Rocky Enterprise Software Foundation - Release key 2022 <releng@rockylinux.org>"
 Fingerprint: 21CB 256A E16F C54C 6E65 2949 702D 426D 350D 275D
 From       : /etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-9

Installed:
  dbus-libs-1:1.12.20-7.el9_2.1.x86_64           dnf-plugins-core-4.3.0-5.el9_2.noarch 
  python3-dateutil-1:2.8.1-6.el9.noarch          python3-dbus-1.2.18-2.el9.x86_64      
  python3-dnf-plugins-core-4.3.0-5.el9_2.noarch  python3-six-1.15.0-9.el9.noarch       
  python3-systemd-234-18.el9.x86_64             

[root@117abf8b612e /]# dnf config-manager --dump-variables
arch = x86_64
basearch = x86_64
rltype = 
stream = 9-stream
infra = container
sigcontentdir = pub/sig
contentdir = pub/rocky
releasever = 9
```

Reply:
Ah, thanks! So, if I interpret that correctly, they also follow a rolling release model, so we should consider using `7`, `8` (and no need to look at "minor" releases, as they effectively don't exist), correct?

Reply:
> Ah, thanks! So, if I interpret that correctly, they also follow a rolling release model, so we should consider using `7`, `8` (and no need to look at "minor" releases, as they effectively don't exist), correct?

Yes, updated PR.